runtime/util/rcdump.c

```diff
@@ -96,6 +96,10 @@ static I_32 dumpCallSiteData ( J9PortLibrary *portLib, J9ROMClass *romClass, U_3
 static I_32 dumpStaticSplitSideTable( J9PortLibrary *portLib, J9ROMClass *romClass, U_32 flags);
 static I_32 dumpSpecialSplitSideTable( J9PortLibrary *portLib, J9ROMClass *romClass, U_32 flags);
 static void printMethodExtendedModifiers(J9PortLibrary *portLib, U_32 modifiers);
+
+BOOLEAN isRomClassValueType(J9ROMClass *romClass) {
```

Comment:
I believe macro `J9ROMCLASS_IS_VALUE()` needs to be updated to include the version checks. You can use `J9ROMCLASS_IS_VALUE()` in this file instead of adding this new function.

Reply:
I don't see the major and minor version checks added to `J9ROMCLASS_IS_VALUE()`.